src/@next/Icon/icons/ri-search.svg

```diff
@@ -0,0 +1,10 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
can you try and check whether passing the fill prop renders the color? I think we need to remove the `fill="none"` in svg for it to work 🤔

Reply:
it will, I did that previously, and that technique is used in our current icons.